cmd/erasure-decode.go

```diff
@@ -189,16 +193,16 @@ func (p *parallelReader) Read(dst [][]byte) ([][]byte, error) {
 	wg.Wait()
 	if p.canDecode(newBuf) {
 		p.offset += p.shardSize
-		if atomic.LoadInt32(&missingPartsHeal) == 1 {
+		if missingPartsHeal == 1 {
```

Comment:
Yeah, but `wg.Wait()` should ensure that it is no longer written to, so it can be accessed without atomics.

Reply:
> Yeah, but wg.Wait() should ensure that it is no longer written to, so it can be accessed without atomics.

I tend to remove extra code to avoid useless confusion